app/src/main/res/values/strings.xml

```diff
@@ -38,6 +38,9 @@
     <string name="initial_count">0</string>
     <string name="plus_button_description">Plus Button</string>
     <string name="team_2">Team 2</string>
+    <string name="ready_to_work">I am ready to start work!</string>
+    <string name="start_task">Start Task</string>
+    <string name="napping">Napping...</string>
     <!-- Placeholder text for this exercise 6. -->
 
```

Comment:
<string name ="text_result_xxx">Awake at last after sleeping for %s milliseconds!</string>